worker.go

```diff
@@ -70,35 +70,35 @@ func (w *Worker) ProcessMetric(m *Metric) {
 			log.WithField("name", m.Name).Debug("New counter")
 			w.counters[m.Digest] = NewCounter(m.Name, m.Tags)
 		}
-		w.counters[m.Digest].Sample(m.Value, m.SampleRate)
+		w.counters[m.Digest].Sample(m.Value.(float64), m.SampleRate)
```

Comment:
It seems kinda displeasing that we don't do checked conversions here, but I guess we can guarantee that this is fine? I guess the thing that would work better here might be some sort of interface method to return the converted value?